t/core/request.t

```diff
@@ -439,3 +439,47 @@ while ($i <= 101) {
 $s
 --- response_body
 101
+
+
+
+=== TEST 15: add header
+--- config
+    location /t {
+        content_by_lua_block {
+            local core = require("apisix.core")
```

Comment:
It is better to use `local json = require("toolkit.json")`, because it will sort the keys of the table

Reply:
Thanks, fixed.